observant-otters/forumthing/user/models.py

```diff
@@ -1,3 +1,14 @@
 from django.db import models
+from django.contrib.auth.models import User
 
-# Create your models here.
+class OurUser(models.Model):
```

Comment:
Instead of writing a seperate model to augment the existing User model, is there some way to extend the User model?

Reply:
Yeah, I was thinking of doing that- will do.